apps/sms/js/thread_list_ui.js

```diff
@@ -498,7 +505,9 @@ var ThreadListUI = {
     var removeWhenEmpty = empty ? 'remove' : 'add';
 
     ThreadListUI.noMessages.classList[removeWhenEmpty]('hide');
+    ThreadListUI.settingsButton.classList[removeWhenEmpty]('hide');
     ThreadListUI.container.classList[addWhenEmpty]('hide');
+    ThreadListUI.optionsButton.classList[addWhenEmpty]('hide');
```

Comment:
> @azasypkin maybe you'll have a better idea than me for the class name? I feel like is-empty is too small and thus too generic.

Yeah, ```is-empty``` is a way too generic, totally agree.

> But to be BEM-like we should add a "ThreadList" class and a "isEmpty" class to have a selector ".ThreadList.isEmpty". What do you think?

We already have ```threadlist``` prefix for other ThreadListUI-related selectors (eg. [1]) so I think it's good to have one. So either ```threadlist-is-empty``` or ```threadlist-no-messages``` sounds good. I'm also fine if we eventually start using BEM-like state modifiers - ```threadlist--empty``` as we discussed in [2]. 

[1] https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/style/sms.css#L144
[2] https://etherpad.mozilla.org/sms-code-style-guidelines